### DISCUSSION TITLE
Make lists expandable

### DISCUSSION THREAD

**Discussion by sbikandi-at-wiris** [community/community#113511] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

It would be great if the lists could be expanded like dropdown or accordion.

**Comment by WojasKrk** [comment]
> ### Select Topic Area
> 
> Product Feedback
> 
> ### Body
> 
> It would be great if the lists could be expanded like dropdown or accordion.

**Reply by WojasKrk** [reply]
> > ### Select Topic Area
> > 
> > Product Feedback
> > 
> > ### Body
> > 
> > It would be great if the lists could be expanded like dropdown or accordion.
> 
>
